packages/forc/VERSION

```diff
@@ -1 +1 @@
-0.49.3
+0.51.1
```

Comment:
It'd be good to check if this works with `forc` @  `0.53.0`:
 - https://github.com/FuelLabs/fuels-ts/pull/2045

This version includes many performance improvements and bug fixes regarding the new encoding.

Keep in mind that, speaking of the new encoding, we will still need this before the end:
 - https://github.com/FuelLabs/fuels-ts/pull/2044

Reply:
@arboleya I've tried `0.55.0` and it is working fine